templates/todo/common/infra/bicep/app/api-appservice-dotnet.bicep

```diff
@@ -10,7 +10,7 @@ param appSettings object = {}
 param keyVaultName string
 param serviceName string = 'api'
 
-module api '../../../../../common/infra/bicep/core/host/appservice.bicep' = {
+module api 'br:publicazdbicepmodules.azurecr.io/bicep/staging/core/host/appservice:v1.0' = {
```

Comment:
I think we should move to semver

Reply:
The format for the tag is not really relevant.  We can change it at any time, for any repository.
There won't be anything on the registry to validate the tag is `semver`.
For the production registry, we can limit the access for publishing modules to some app which would integrate the `semver` validation, but this is orthogonal to this PR.

Reply:
It is critical that we figure out how we version modules and push those versions out to new templates.  I don't mind if you do it in a different PR, but figuring that out is a P1 to ship this.